src/quemb/shared/external/dmrgscf_settings.py

```diff
@@ -0,0 +1,17 @@
+#
+# This file sets the environment.
+# Configs will be used in dmrgsci.py and chemps2.py
+#
+
+import os
+
+# For Block and Block2 solvers
+BLOCKEXE = "block2main"
+BLOCKEXE_COMPRESS_NEVPT = "block2main"
+BLOCKSCRATCHDIR = str(os.getpid())
+BLOCKRUNTIMEDIR = str(os.getpid())
```

Comment:
I don't know how `block` is called, but shouldn't these be directories, not PIDs?